scripts/bootstrap-aio.yml

```diff
@@ -62,6 +62,16 @@
         - name: swift.yml.aio
         - name: ceph.yml.aio
           path: "{{ lookup('env', 'RPCD_DIR') ~ '/etc/openstack_deploy/conf.d' }}"
+      ironic:
+        - name: cinder.yml.aio
```

Comment:
Are cinder, horizon and heat needed?

Reply:
We can create a task to verify that

Reply:
let me know when it's created :P

Reply:
https://rpc-openstack.atlassian.net/secure/RapidBoard.jspa?rapidView=18&projectKey=RI&view=detail&selectedIssue=RI-88